CREATE-NEW-SCALER.md

```diff
@@ -12,8 +12,10 @@ In order to developer a scaler, a developer should do the following:
 6. Run `make build` from the root of KEDA and your scaler is ready.
 
 If you want to deploy locally 
-1. Open the terminal and go to the root of the source code, then build a docker image of KEDA by running `docker build . -t [choose a unique tag for your custom image]`
-2. In the terminal, navigate to the `chart/keda` folder, and run the following command (don't forget to replace the placeholder text in the command) `helm install . --set image.repository=[tag used in step 1],image.pullPolicy=IfNotPresent`.
+1. run: `export IMAGE_TAG=yourchosenname`
```

Comment:
Let's make it a bit more clear
```suggestion
1. Run `export IMAGE_TAG=local`
```

Reply:
For some reason i can't commit your proposed changes (last time i was able to do it), so probably if you can just commit them yourself (but just remember to update also the numbering) :)